src/transformers/searchservice/createDirectAnswer.ts

```diff
@@ -25,7 +25,11 @@ export function createDirectAnswer(data: any): FeaturedSnippetDirectAnswer | Fie
     };
   } else if (isFeaturedSnippetDirectAnswer) {
     const fieldType = commonDirectAnswerData.fieldType;
-    if (fieldType != BuiltInFieldType.MultiLineText && fieldType != BuiltInFieldType.RichText) {
+    if (fieldType != BuiltInFieldType.MultiLineText &&
+        fieldType != BuiltInFieldType.RichText &&
+        fieldType != BuiltInFieldType.RichText_v2 &&
+        fieldType != BuiltInFieldType.Html &&
+        fieldType != BuiltInFieldType.Markdown) {
```

Comment:
nit: can you put these built in field types in a list and call the `includes` method instead of checking each one individually?